README.md

```diff
@@ -1,67 +1,146 @@
-
-### Manual Setup
-```bash
-$ npm install
-```
-
-Create a file within the repo folder name `.env` with content:
-```.env
-DISCORD_TOKEN={your-discord-bot-token}
-CLIENT_ID={your-discord-client-id}
-```
-
-Replace `'{your-discord-bot-token}'` with your Discord bot token
-
-## Development
-After you've completed the setup you should be able to run
-```bash
-$ node index.js
-```
-This will run the bot locally on your machine which will connect to any guilds you've set it to connect to within [Discord's Developer Portal](http://discordapp.com/developers/applications) for the bot. (See: [How to Make a Discord Bot in the Developer Portal](https://realpython.com/how-to-make-a-discord-bot-python/#how-to-make-a-discord-bot-in-the-developer-portal))
-
-## Registering Slash (/) Commands
-```bash
-$ node registerSlashCommands.js
-```
-Running this will register all slash commands under `/commands` to the guilds (servers) who have IDs within the `guildIDs` array within the `registerSlashCommands.js` file.
-
-Slash commands are only included in DiscordJS v13.x.x or higher. For more information on slash (/) commands and registering them see [Discord's documentation](https://discordjs.guide/interactions/registering-slash-commands.html#guild-commands).
-
----
-
-### ideas/todo
-From previous Python bot ->
-1. accrue tokens with activity
-2. distribute tokens person to person
-3. records log of every bet and distribution of tokens
-4. *salsa*
-5. register different kinds of bets (rng, bets regarding users, bets regarding collectives)
-6. **guac**
-7. algorithm that determines odds payouts based on bet sizes for each outcome, or even use data to determine likelihood of an event (use porofessor to retrieve winrates, etc)
-8. distribute base amount of tokens
-9. cash in tokens for discord/gaming privileges/punishments
-10. roll_deez
-11. roll_meez_shinz
-12. quotes
-13. ask for quote of specific user
-14. after for random or numbered quote
-
-From after creation of this bot ->
-1. consolidate audio and voice connect related code within `utils/connectAndPlayAudioFile.js` and `utils/speakText`
-2. set up music bot capabilites with queueing, pausing, stopping, etc. 
-    - potentially take all music bot code and put into a separate bot so as to avoid voice-overlap. alternatively, have the bot not do anything voice related while playing audio/music.
-3. set audio recording capabilities
-4. add constants.js with constant definitions
-5. change /playaudio and /listaudio to /audio play and /audio list subcommands
\ No newline at end of file
+# PASTRAMI BOT
+
+[![GitHub issues](https://img.shields.io/github/issues/jacobc18/js-discord-bot)](https://github.com/jacobc18/js-discord-bot/issues)
+
+![hello there](https://media.giphy.com/media/3ornk57KwDXf81rjWM/giphy.gif)
+
+
+
+
+Pastrami is a multifunctional JavaScript Discord bot with abilities including
+
+- TTS greetings for voice channel user join
+- Audio streaming via YouTube links 
+- Information on famous bald people

```

Comment:
Let's add more of the capabilities like these:
- playing local audio files
- performing TTS from given strings
(feel free to make the wording better however you like)